src/script/interpreter.h

```diff
@@ -98,13 +98,20 @@ enum
 
 bool CheckSignatureEncoding(const std::vector<unsigned char> &vchSig, unsigned int flags, ScriptError* serror);
 
+struct CachedHashes
```

Comment:
It's actually not a cache either anymore.

What about PrecomputedTransactionData ?